dev-python/icalendar/icalendar-4.0.6.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit distutils-r1
+
+DESCRIPTION="Package used for parsing and generating iCalendar files (RFC 2445)"
+HOMEPAGE="https://github.com/collective/icalendar"
+SRC_URI="mirror://pypi/i/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="amd64 ~arm ~arm64 x86"
+DOCS=( "README.rst" )
+
+RDEPEND="
+	dev-python/python-dateutil[${PYTHON_USEDEP}]
+	dev-python/pytz[${PYTHON_USEDEP}]"
+
+BDEPEND="
```

Comment:
All duplicate.

Reply:
If I understand correctly, you mean that `RDEPEND` is added to `BDEPEND` when `USE=test` automatically by `distutils_enable_tests`, and it doesn’t need to be there when `USE=-test`; and that `dev-python/sphinx` is added automatically by `distutils_enable_sphinx`. So that means `BDEPEND` should be completely unecessary. Deleted in d98f38aa7cce.